clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -604,7 +604,6 @@ export const CustomSelect = ({
                 isCustomOption={isCustomOption}
                 singleValueBlock={singleValueBlock}
                 menuPosition={props.menuPosition}
-                onChange={!isFormikOnChange ? onChange : undefined}
```

Comment:
I believe you removed the wrong on change or still need to remove line 650 https://github.com/ethyca/fides/pull/4327/files#diff-f59681eb1681d3b25307f6abef05b10c5926bafe6543643fcb07a95e5cd4e55fR650